### ISSUE TITLE
Blank page between [ACNEMED3] and [FAMHISTINTRO]

### ISSUE THREAD

**Issue by cunnaneaq** [episphere/connectApp#334]
![Screen Shot 2022-04-08 at 10 05 16 AM](https://user-images.githubusercontent.com/83971268/162452135-2991ba81-a511-475d-ae83-f0e074d3938c.png)

![Screen Shot 2022-04-08 at 10 05 26 AM](https://user-images.githubusercontent.com/83971268/162452171-54856c2d-8213-40bf-80a7-c735c4dca514.png)

![Screen Shot 2022-04-08 at 10 05 41 AM](https://user-images.githubusercontent.com/83971268/162452221-09dc24d1-c026-4cca-b8c2-321139882e40.png)

**Comment by anthonypetersen** [comment]
is this still an issue?

**Comment by cusackjm** [comment]
@anthonypetersen fixed

**Comment by cusackjm** [comment]
@anthonypetersen still present in stage. Is this fixable, or should we add the 'Thank you for responding, please continue' message that IMS added to other blank pages that couldn't be fixed

**Comment by anthonypetersen** [comment]
@danielruss thoughts? on the surface doesn't feel like something that just "happens"...

**Comment by danielruss** [comment]
@anthonypetersen  Does the module end with the the [END] question?

**Comment by anthonypetersen** [comment]
@danielruss Aileen and Julie just showed me how to recreate this.. They got into a loop by answering "YES" to question 512167179, then after answering question 967093163 the next question pointed to something called "SECTION4" which was the blank screen. You were still able to continue on after that point as normal so this isn't "breaking" anything.

**Comment by danielruss** [comment]
So section4 is a blank question?

**Comment by anthonypetersen** [comment]
![image](https://user-images.githubusercontent.com/18222069/186259047-b3dd50d1-e965-4f2d-b50b-82594aa2d249.png)

**Comment by anthonypetersen** [comment]
looking at that logic I'm not sure why line 1225 would lead to line 1227 instead of 1229 @danielruss

**Comment by danielruss** [comment]
I agree.  The blank page is caused by the blank question.  Ask if you can delete either the question called SECTION4 or FAMHISTINTRO.

**Comment by cusackjm** [comment]
I would prefer deleting SECTION4 over FAMHISTINTRO

**Comment by anthonypetersen** [comment]
what's the point of SECTION4?

**Comment by danielruss** [comment]
I agree with @cusackjm.

**Comment by cusackjm** [comment]
@anthonypetersen not sure. there is no contextual significance for the participant
